Search-algorithms/src/base/Bubblesort.java

```diff
@@ -0,0 +1,19 @@
+package base;
+
+public class Bubblesort {
+	
+public static int i,j,tmp;
```

Comment:
invece di usare memoria fisica per queste variabili potresti dichiararle all'interno del metodo in cui le usi